cli/tests/conftest.py

```diff
@@ -88,6 +94,10 @@ def boto3_stubber(mocker, boto3_stubber_path):
     mocked_client_factory.client.side_effect = lambda x, **kwargs: mocked_clients[x]
 
     def _boto3_stubber(service, mocked_requests):
+        if "AWS_DEFAULT_REGION" not in os.environ:
+            # We need to provide a region to boto3 to avoid no region exception.
+            # Which region to provide is arbitrary.
+            os.environ["AWS_DEFAULT_REGION"] = "us-east-1"
```

Comment:
I'd avoid setting the region in the boto3_stubber fixture.

We should instead mock the region in the tests using it, with the right region used by the test